package.json

```diff
@@ -95,18 +95,18 @@
     "@types/jest": "^27.0.0",
     "@types/node": "^14.0.0",
     "@types/prettier": "^2.0.0",
-    "@typescript-eslint/eslint-plugin": "^4.0.1",
-    "@typescript-eslint/parser": "^4.0.1",
+    "@typescript-eslint/eslint-plugin": "^5.0.0-0",
+    "@typescript-eslint/parser": "^5.0.0-0",
     "babel-jest": "^27.0.0",
     "babel-plugin-replace-ts-export-assignment": "^0.0.2",
     "dedent": "^0.7.0",
-    "eslint": "^6.0.0 || ^7.0.0",
+    "eslint": "^6.0.0 || ^7.0.0 || ^8.0.0-0",
     "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-eslint-comments": "^3.1.2",
+    "eslint-plugin-eslint-comments": "^3.2.0",
     "eslint-plugin-eslint-config": "^2.0.0",
     "eslint-plugin-eslint-plugin": "^3.5.3",
-    "eslint-plugin-import": "^2.20.2",
-    "eslint-plugin-node": "^11.0.0",
+    "eslint-plugin-import": "^2.24.2",
+    "eslint-plugin-node": "^11.1.0",
```

Comment:
Seems like it doesn't make any real change

Reply:
Will update this to latest whenever these dependencies support ESLint v8

Reply:
sure, but for now the version change does nothing since they are in semver range. So it's just extra noise